plugins/org.eclipse.reddeer.core/META-INF/MANIFEST.MF

```diff
@@ -18,3 +17,7 @@ Export-Package: org.eclipse.reddeer.core.condition,
  org.eclipse.reddeer.core.reference,
  org.eclipse.reddeer.core.resolver,
  org.eclipse.reddeer.core.util
+Import-Package: org.eclipse.reddeer.workbench.core.lookup;resolution:=optional,
```

Comment:
Is there any situation were the package is not needed? Moreover, get back the version range [2.1, 2.2)

Reply:
Yes, if you want to test SWT app (no Workbench)